apps/infrastructure/src/modules/chatbot/dynamodb.tf

```diff
@@ -41,5 +41,19 @@ module "dynamodb_chatbot_sessions" {
       name = "userId"
       type = "S"
     },
+    {
+      name = "createdAt"
+      type = "S"
+    },
   ]
+
+  # GSI for query on created_at
+  global_secondary_index {
+    name            = "SessionsByCreatedAtIndex"
+    hash_key        = "userId"
+    range_key       = "createdAt"
+    read_capacity   = 5
+    write_capacity  = 5
```

Comment:
Should be removed since we are not using the PROVISIONED billing_mode
```suggestion
```

Reply:
done